picard/tagger.py

```diff
@@ -318,12 +318,11 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate, pipe_handl
             self.updatecheckmanager = UpdateCheckManager(parent=self.window)
 
     def pipe_server(self):
-        IGNORED = {pipe.Pipe.MESSAGE_TO_IGNORE, pipe.Pipe.NO_RESPONSE_MESSAGE}
+        IGNORED = {pipe.Pipe.MESSAGE_TO_IGNORE, pipe.Pipe.NO_RESPONSE_MESSAGE, ""}
```

Comment:
When does `""` occur?

In `read_from_pipe()` `if r` should prevent it to happen.